test/postgresql.test.js

```diff
@@ -230,6 +230,111 @@ describe('postgresql connector', function() {
     });
   });
 
+  context('pattern matching operators', function() {
+    before(function deleteTestFixtures(done) {
+      Post.destroyAll(done);
+    });
+    before(function createTextFixtures(done) {
+      Post.create([{
+        title: 't1',
+        content: 'T1_TestCase',
+      }, {
+        title: 't2',
+        content: 'T2_TheOtherCase',
+      }], done);
+    });
+    after(function deleteTestFixtures(done) {
+      Post.destroyAll(done);
+    });
```

Comment:
@superkhau Thanks for the review. I will apply these changes, rebase, and push the changes in the next few minutes.